### ISSUE TITLE
Fold background and foreground color different from the one set in hi Folded group

### ISSUE THREAD

**Issue by paradoxsupreme** [lervag/vimtex#1851]
The fold background and background color is different from the one set in hi Folded group.

**Comment by lervag** [comment]
Ehm, no, it isn't?

Please share a minimal setup; if you can make me reproduce your problem I'll remove my claim.

**Comment by paradoxsupreme** [comment]
> Ehm, no, it isn't?
> 
> Please share a minimal setup; if you can make me reproduce your problem I'll remove my claim.
Hmm let me try again then, maybe it has something to do with my settings because I tried without any plugins except from vimtex. If it will not be solved then I will post again...

**Comment by lervag** [comment]
Sounds good. Since you say you might not post again, I'll take the liberty of closing the issue. Feel free to reopen; I will read your updates in any case.

**Comment by paradoxsupreme** [comment]
I found what was causing this. It happens when I enable spell checking.

**Comment by lervag** [comment]
And you're on neovim, I assume; also on a recent build? If so, then this is a bug in neovim. Sorry :\

**Comment by lervag** [comment]
https://github.com/neovim/neovim/issues/12982
